kahypar/partition/recursive_bisection.h

```diff
@@ -233,6 +234,12 @@ static inline void partition(Hypergraph& input_hypergraph,
       HypergraphPtr(hg_without_fixed_vertices.first.release(),
                     delete_hypergraph);
     fixed_vertex_free_to_input = hg_without_fixed_vertices.second;
+
+    for ( PartitionID block = 0; block < original_context.partition.k; ++block ) {
```

Comment:
@kittobi1992 I might be missing something, but shouldn't this be something like

```c++
if (original_context.use_individual_part_weights) {
  for ( PartitionID block = 0; block < original_context.partition.k; ++block ) {
     rb_context.partition.max_part_weights[block] -= input_hypergraph.fixedVertexPartWeight(block);
  }
} 
rb_context.setupPartWeights(input_hypergraph_without_fixed_vertices->totalWeight());
```
so that we use the "normal" block-weight setup code whenever we don't use individual part weights?

Reply:
No, the fixed vertex block weights can be different for each block, which leads to individual block weights for initial Partitioning. If we do not use individual block weights, the part weight setup function will assign the same max part weight for each block.

Reply:
Ah, I see. That makes sense. Thanks for your explanation.